composer.json

```
@@ -301,7 +301,8 @@
             "Rector\\Utils\\NodeDocumentationGenerator\\Tests\\": "utils/node-documentation-generator/tests",
             "Rector\\Utils\\PHPStanTypeMapperChecker\\": "utils/phpstan-type-mapper-checker/src",
             "Rector\\Utils\\ProjectValidator\\": "utils/project-validator/src",
-            "Rector\\Carbon\\Tests\\": "rules/carbon/tests"
+            "Rector\\Carbon\\Tests\\": "rules/carbon/tests",
+            "Rector\\PostRector\\Tests": "packages/post-rector/tests"
```

need namespace separator:

```suggestion
            "Rector\\PostRector\\Tests\\": "packages/post-rector/tests"
```

---

Yeah 👍